samples/bluetooth/throughput/README.rst

```diff
@@ -111,8 +111,8 @@ After programming the sample to both kits, complete following steps to test it:
 
 1. |connect_terminal_both_ANSI|
 #. Reset both kits.
-#. Press **Button 1** on the kit to set the kit into central (tester) role.
-#. Press **Button 2** on the other kit to set the kit into peripheral (peer) mode.
+#. Press **Button 1** or type ``central`` in the terminal on the kit to set the kit into central (tester) role.
+#. Press **Button 2** or type ``peripheral`` in the terminal on the other kit to set the kit into peripheral (peer) mode.
```

Comment:
```suggestion
#. Press **Button 2** on the second development kit or type ``peripheral`` in the terminal on the other kit to set it into the peripheral (peer) role.
```
Is it mode or role?

Reply:
role

Reply:
OK, so please use my suggestion's edit :)